pkgs/build-support/rust/build-rust-package/default.nix

```diff
@@ -150,6 +151,11 @@ stdenv.mkDerivation ((removeAttrs args [ "depsExtraArgs" "cargoUpdateHook" "carg
 
   strictDeps = true;
 
+  postInstall = lib.optionalString isDarwinDebug ''
+    cp -RL ./target/${cargoInstallHook.targetSubdirectory}/debug/*.dSYM $out/bin/
+  '';
+  RUSTFLAGS = (args.RUSTFLAGS or "") + lib.optionalString isDarwinDebug "-C split-debuginfo=packed";
```

Comment:
Maybe this should be part of RUSTFLAGS in line 100?

Reply:
I mean if sysroot is configured and than also debug both, the sysroot introduced RUSTFLAGS gets ignored

Reply:
I had tried that earlier and kept getting weird build errors about `/private/etc/ssl/openssl.cnf`. Perhaps sandbox related (and perhaps resolved with @emilazy's recent changes?) so will try again.

Reply:
Hmmm... https://gist.github.com/n8henrie/1b9f3cc63260edf844c41f78e8bd8423

Reply:
How is that? I never see all-caps arguments, so I imagine it might be a style issue or perhaps a "`substitute` needs a lower-case initial letter" issue.

Reply:
Hmmm, might be a problem with sysroot. I need to read more about what sysroot is all about (something about `no_std` development?).